packages/components/src/text/stories/index.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Internal dependencies
+ */
+import { __experimentalText as Text } from '../text.styles';
+
+export default {
+	title: 'Components|Text',
```

Comment:
Should it be marked as experimental here as well?

Reply:
Sure! Whats the convention?

Reply:
There is no prior art, you can propose one 😃

Reply:
Well, I have an idea:

```suggestion
	title: 'Components|Experimental/Text',
```

I thin @ItsJonQ used this syntax for `Card` components. It groups them together in the subsection.

Still, if you have a better idea, just go with it.